deps/wazuh_testing/wazuh_testing/logcollector.py

```diff
@@ -1,14 +1,22 @@
+import sys
+
 from wazuh_testing.tools import monitoring
+from wazuh_testing.tools.monitoring import AGENT_DETECTOR_PREFIX, LOG_COLLECTOR_DETECTOR_PREFIX
```

Comment:
```suggestion
from wazuh_testing.tools import monitoring
```

Reply:
Fixed in https://github.com/wazuh/wazuh-qa/commit/e64facee7f700035889c107f932edc537449fec2